src/components/tabs/js/tabsController.js

```diff
@@ -655,7 +655,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
     var elements = getElements(),
       containerWidth = elements.canvas.clientWidth,
 
-      // See https://material.io/design/components/tabs.html#spec which has been updated to 360px.
+      // See https://material.io/archive/guidelines/components/tabs.html which has been updated to 360px.
```

Comment:
Let's change this to
`// See https://material.io/archive/guidelines/components/tabs.html#tabs-specs`
and remove the other part about `360px` since that refers to the 2018 spec.

Reply:
This has been updated :chicken: